server/src/controllers/users.js

```diff
@@ -72,7 +72,6 @@ module.exports = {
 
 	getRuns: (req, res, next) => {
 		req.query.playerID = req.params.userID;
-		req.query.order = 'date';
```

Comment:
I'm not sure the consequences of doing this but honestly if the default runMdl.getAll just sorts by date anyways this would be fine. Would be worth looking into what all uses `runMdl.getAll` and see if they rely on the order being `ticks`.

Reply:
It seems to only be used [here](https://github.com/momentum-mod/website/blob/646f8ab8f3eeb6d85d333c2b8266165166b6c4df/server/src/controllers/runs.js) elsewhere. I could also re-add the default query order parameter for date and override it only if an order parameter is issued to the service if that's more clear/safe.

Reply:
On further review only the runs service seems to call `/api/runs/` from the aforementioned link and nothing seems to call the runs service `getRuns`. Nevertheless I've changed it back to be like the old result on default return in case something in the future relies on `getRuns` returning in time order. I did, however, make it return in time order instead of tick order since I presume that was what was expected from before.